### ISSUE TITLE
Similar Experience - Git Source Code Repository

### ISSUE THREAD

**Issue by pnedungottil** [18F/bpa-opm-eqip#56]
# Question/Comment on e-QIP Prototype RFQ

## Name and affiliation
Praveen Nedungottil - VP, Ventera Corp

## Section of RFQ documents
eQIP RFQ Section 6.f

## Question/Comment
Is it ok to provide a link and credentials for accessing a private Git source code repository (as opposed to a public Git)?

**Comment by jehlers** [comment]
No. Any information considered as part of the evaluation must be reviewable by a third party, and therefore must be accessible, either publicly or through documents provided to the evaluation panel.
